test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -219,6 +219,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.cloud.organization_url (string)',
         'xpack.cloud.billing_url (string)',
         'xpack.cloud.profile_url (string)',
+        'xpack.cloud.serverless.project_id (string)',
```

Comment:
__optional nit:__ it'd be great to leave a comment saying that it doesn't include anything sensitive, just in case we have to conduct an audit of this file at some point.